compact_proof/scripts/install_prover.sh

```diff
@@ -2,9 +2,4 @@
 
 set -eoux
 
-docker build -f docker/build.Dockerfile . -t snark-build --output=groth16 --target outputs
-
-wget https://risc0-artifacts.s3.us-west-2.amazonaws.com/zkey/2024-02-07/stark_verify_final.zkey.gz -O groth16/stark_verify_final.zkey.gz
-(cd groth16; gzip -df stark_verify_final.zkey.gz)
-
 docker build -f docker/prover.Dockerfile . -t risc0-groth16-prover
```

Comment:
I would love if we could publish this docker image into our CI s3 bucket or ECR so we don't have to git clone the whole `risc0` repo to access the source of truth build of this docker image.

Reply:
Yes I agree with that! 
I feel like we have a few options:
1. publish on ECR
2. publish on GHCR
3. publish on Docker Hub

My favourite is the 3. but I might be the minority here. The the build should only be done every time we update the ceremony (i.e., we re-run the xtsask bootstrap-groth16), so not that often.  And providing the docker image publicly would also make CI faster during tests as well as faster for users to potentially run locally.

Overall though I'm open to any of the above. Docker save is also nice but not as clean imo

Reply:
I would say, option 1, 3 and 4 (docker save + s3) are a best for us. We can probably make any of those options work really easily.

Reply:
Option 3 and 4 are nice because it is easiest for external folks to pull the image as well and extend it.